File: src/test/resources/go/MathOperations.go
package main

import "fmt"

type MathOperations struct{}

func (self MathOperations) calculateResult1(x, y int) int {
	sum := x + y
	product := x * y
	difference := x - y
	return (sum + product - difference) + 10
}

func (self MathOperations) calculateResult2(p, q int) int {
	difference := p - q
	product := p * q
	sum := p + q
	return 2 * (sum + product - difference)
}

File: src/test/resources/kt/MathOperations.kt
class MathOperations {
    fun calculateResult1(x: Int, y: Int): Int {
        val sum = x + y
        val product = x * y
        val difference = x - y
        return (sum + product - difference) + 10
    }

    fun calculateResult2(p: Int, q: Int): Int {
        val difference = p - q
        val product = p * q
        val sum = p + q
        return 2 * (sum + product - difference)
    }
}
